xdsl/dialects/riscv.py

```diff
@@ -704,6 +1052,321 @@ class LiOp(RdImmOperation):
     name = "riscv.li"
 
 
+@irdl_op_definition
+class LBOp(RdRsImmOperation):
+    name = "riscv.lb"
```

Comment:
This and the obs below are already upstream. I feel the rebase / merge accidentally copied these operations. Maybe drop the duplicated ones to make the diff smaller.

Reply:
Indeed, will drop